docs/guide/vec_envs.rst

```diff
@@ -90,3 +90,15 @@ VecTransposeImage
 
 .. autoclass:: VecTransposeImage
   :members:
+
```

Comment:
as a general remark, as mentioned in the PR template and contributing guide, please update the changelog accordingly too